From: Bastian Blank <waldi@debian.org>
Subject: DocBook: Make Documentation/DocBook -j clean
Date: Tue, 14 Jun 2006 00:05:06 +0200

Two concurrent calls to cmd_db2man may attempt to compress manual
pages generated by each other.  gzip can then fail due to an input
file having already been compressed and removed.

Since dh_compress will compress manual pages later, we don't need
to run gzip here at all.

Index: linux-sunxi-3.0.42/Documentation/DocBook/Makefile
===================================================================
--- linux-sunxi-3.0.42.orig/Documentation/DocBook/Makefile	2012-11-09 09:10:54.000000000 +0000
+++ linux-sunxi-3.0.42/Documentation/DocBook/Makefile	2012-11-12 11:42:53.068527966 +0000
@@ -170,7 +170,7 @@
             cp $(PNG-$(basename $(notdir $@))) $(patsubst %.html,%,$@); fi
 
 quiet_cmd_db2man = MAN     $@
-      cmd_db2man = if grep -q refentry $<; then xmlto man $(XMLTOFLAGS) -o $(obj)/man $< ; gzip -f $(obj)/man/*.9; fi
+      cmd_db2man = if grep -q refentry $<; then xmlto man $(XMLTOFLAGS) -o $(obj)/man $< ; fi
 %.9 : %.xml
 	@(which xmlto > /dev/null 2>&1) || \
 	 (echo "*** You need to install xmlto ***"; \
